guides/common/modules/proc_upgrading-packages-on-a-host.adoc

```diff
@@ -1,14 +1,28 @@
 [id="upgrading-packages-on-a-host_{context}"]
 = Upgrading Packages on a Host
 
-You can upgrade packages on a host in bulk in the {ProjectWebUI}.
+You can upgrade a single package or multiple packages on a host in the {ProjectWebUI}.
+The package upgrade process takes place via Katello agent or remote execution, depending on your configuration.
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
 . Click the name of the host you want to modify.
 . Click the *Content* tab, then click the *Packages* tab.
 . Select *Upgradable* from the *Status* list.
-. Select upgrade version from the dropdown menu in *Upgradable to* column where applicable.
+. Select an upgrade version from the dropdown menu in *Upgradable to* column where applicable.
 . Select the packages you want to upgrade.
-. Click the *Upgrade* button.
+. Click *Remote Execution* or *Customized Remote Execution* from the vertical ellipsis on the right of the package.
+** To upgrade multiple packages at a time, select the *Upgradable* filter from the *Status* box to list the upgradable packages.
+
+You can run a remote execution job immediately, or select to customize the remote execution job.
+
+[NOTE]
+====
+If a package has more than one available upgrade version, only the latest upgradeable version is displayed.
+ifdef::katello[]
+This behavior has been replaced by a dropdown list of all updates.
+Please check the Katello version when it is changed.
```

Comment:
Are you seriously telling the user to check for themself when the feature was added? It is our responsibility to describe features correctly in each version of the docs.

Additionally, the Katello version always corresponds with a particular Foreman version as you can see in the [PR template](https://github.com/theforeman/foreman-documentation/blob/master/.github/PULL_REQUEST_TEMPLATE.md).

Reply:
Removed the note.